include/nlohmann/detail/macro_scope.hpp

```diff
@@ -112,10 +112,10 @@
              template<typename, typename...> class ArrayType,              \
              class StringType, class BooleanType, class NumberIntegerType, \
              class NumberUnsignedType, class NumberFloatType,              \
-             template<typename> class AllocatorType,                       \
+             class BinaryType, template<typename> class AllocatorType,     \
```

Comment:
Just to be sure: adding the `BinaryType` not to the end of the list would be a breaking change, right?

Reply:
Yes, I can move it to the back of the list though, it was honestly something I hadn't considered.

Reply:
No worries. But let's move it to the back not to break existing code. We can move it to a better position once we go for 4.0.0.